enhancements/storage/manila-csi-driver-operator.md

```diff
@@ -94,37 +94,37 @@ Actions:
 
 - Go through the lifecycle of volume (create, update, mount, write data, delete, etc.)
 
-#### Write Manila CSI driver operator
+#### Write Manila CSI driver operator (Done)
 
 The operator should be able to create, configure and manage Manila CSI driver for Kubernetes and OpenShift. In other words, automate what has been done on the previous step.
 Actions:
 
-- Create a new repo in OpenShift’s github: github.com/openshift/manila-csi-driver-operator
+- Create a new repo in OpenShift’s github: github.com/openshift/csi-driver-manila-operator
 
-- Implement the driver, using [OpenShift Operator SDK](https://docs.openshift.com/container-platform/4.1/applications/operator_sdk/osdk-getting-started.html)
+- Implement the operator, using [OpenShift Operator SDK](https://docs.openshift.com/container-platform/4.1/applications/operator_sdk/osdk-getting-started.html)
 
 ##### CSI driver operator installation
 
-The installation will include next steps:
+Starting from OpenShift 4.5 the installation will include next steps:
 
 - `cluster-version-operator` starts `openshift-cluster-storage-operator`.
 
-- `openshift-cluster-storage-operator` checks it runs on OpenStack and Manila service is available
+- `openshift-cluster-storage-operator` checks if it runs on OpenStack, starts Manila CSI driver operator and monitors its status (i.e. monitors existence of Manila CSI driver operator ClusterOperator CR and reports its own `openshift-cluster-storage-operator` status based on it).
```

Comment:
It should be explicitly noted that openshift-cluster-storage-operator will install the Manila operator via OLM. Reference to  https://github.com/openshift/enhancements/pull/139 would help (and please track the enhancement closely, as it will impact *all* CSI driver operators, incl. Manila)

Reply:
done